cli/ioctl/cmd/config/config.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package config
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var configFileName string
+
+// ConfigCmd represents the config command
+var ConfigCmd = &cobra.Command{
+	Use:       "config",
+	Short:     "Set or get configuration for ioctl client",
+	ValidArgs: []string{"set", "get"},
+	Args:      cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("Print: " + strings.Join(args, " "))
+	},
+}
+
+func init() {
+	dir := os.Getenv("HOME") + "/.config/iotex"
+	if err := os.MkdirAll(dir, 0700); err != nil {
+		fmt.Println(err.Error())
+		os.Exit(1)
+	}
+	configFileName = dir + "/config.default"
```

Comment:
delete this line

Reply:
the reason is that the dir ~/.config/ioctl/ does not exist yet, so have to create this dir first, then create/open the config file in it
tried directly creating ~/.config/ioctl/config_default , it won't work, that is why code this way